content/articles/index.md

```diff
@@ -0,0 +1,71 @@
+- The reader should know what's an algorithm and how it's used in programming.
+
+- The reader should be familiar with data structures and neural networks.
+
+
+### Introduction
+When playing video games, have you ever wondered how or rather what makes the light in the game look similar to the one in real life? This is because of the *ray tracing technique*, with this process, the game designers can make visible light beam jump into objects, produce real shadows, and create life-like reflections.
+
+Ray tracing has been of great impact in the world today, modifying and innovating various sectors of the tech world. In this article, we'll get to know about ray tracing algorithm in machine learning.
+
+
+### Tables of contents
+- [What is Ray Tracing Algorithm](#what-is-ray-tracing-algorithm)
+- [History of Ray Tracing](#history-of-ray-tracing)
+- [Types of Ray Tracing Algorithm](#types-of-ray-tracing-algorithm)
+  - [Hybrid Ray Tracing Algorithm](#hybrid-ray-tracing)
+  - [Forward Ray Tracing Algorithm](#forward-ray-tracing)
+  - [Backward Ray Tracing Algorithm](#backward-ray-tracing) 
+ - [Conclusion](#conclusion)
+
+
+### What is Ray Tracing Algorithm and how does it work?
+Ray Tracing Algorithm is a very effective way of drawing synthesizing images. It's a method of rendering, this process has the potential to produce a much greater degree of authenticity that is probably higher than that of other scanners. However, this involves a huge calculation cost. Computer imagery has become a very basic need in today's digital world. Computer graphics come into play when we need to create an image that looks real and indistinguishable from the standard image taken. There are many ways to accomplish this. Ray Tracing is one such way to give. This is used to produce an image by tracing the light path such as pixels in an image plane and mimicking its effects when encountering objects.
+
+
+The process aims to bring real light to the simulation by imitating the behavior of light intensity. It is done by reflecting the light rays from the camera, ‘following’ them until they encounter an object, and counting where each ray of light cuts an object in the square. This then allows it to calculate how the pixel is affected by the light rays, and to keep track of the ray as it jumps on the scene. This requires the computer to track a certain number of rays from each intersection to the reflected light model, thus creating a growing number of rays in the square.
+
+
+
+### History of Ray Tracing
+It was created by Robert Goldstein and Arthur Appel in the 1960s. According to [Wikipedia](https://en.wikipedia.org/wiki/Ray_tracing_(graphics)#Algorithm_overview), in 1968, Arthur Appel was the first to use a computer to generate shaded images using ray tracing. He used ray tracing to figure out which surface was closest to the camera at each image point, then traced secondary rays to the source of light from each shaded point to see if it was in shadow or not.

```

Comment:
This history can be placed after the first section (what is ray tracing algorithm)